<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaLogin</class>
 <widget class="QDialog" name="TelaLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SAHD</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Entrar</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>259</width>
      <height>154</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Login">
       <property name="text">
        <string>Registro</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_Registro</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Registro"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Senha">
       <property name="text">
        <string>Senha</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_Senha</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Senha">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_Entrar">
         <property name="text">
          <string>Entrar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Sair">
         <property name="text">
          <string>Sair</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>300</width>
     <height>175</height>
    </rect>
   </property>
   <property name="text">
    <string>LOGO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
